### PR TITLE
OCI Data Flow - Example to load data into ADW

### PR DESCRIPTION
**Pull request by eloilopes**

### CODE REVIEW

**Comment by AlexanderHodicke** [comment]
@oheimburger please see the code - Thanks!

**Comment by eloilopes** [comment]
Any update?

**Review by sylwesterdec — APPROVED** [review]

**Review by sylwesterdec — APPROVED** [review]

File: data-platform/open-source-data-platforms/oci-data-flow/code-examples/load-data-to-adw.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    .load(args.inputpath)
+    .cache())  
+    # cache the dataset to increase computing speed
+
+    df = df.withColumn("DATE_KEY",to_date(df["DATE_KEY"],"dd.MM.yyyy"))
+
+    df.write.format("oracle") \
+    .option("adbId","<autonomous database ocid>") \
+    .option("dbtable", "<table name>") \
+    .option("user", "<username>")\
```
Comment (oheimburger) [inline data-platform/open-source-data-platforms/oci-data-flow/code-examples/load-data-to-adw.py:73]: Do not hard code username and password. Use parameters for command invocation.

**Review by oheimburger — COMMENTED** [review]
Please review my comments.

File: data-platform/open-source-data-platforms/oci-data-flow/code-examples/run-oci-data-flow-from-functions.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                retval['error'] = 'Missing mandatory field ' + item
+                return response.Response(
+                        ctx,
+                        response_data=json.dumps(retval),
+                        headers={"Content-Type": "application/json"})
+
+        applicationId = body.get('applicationId',' <OCI ID application data flow>')
+        compartmentId = body.get('compartmentId','<Compartment ID>')
+        displayName = body.get('displayName','MaterialInventory')
+        #driverShape = body.get('driverShape','VM.Standard2.1')
```
Comment (oheimburger) [inline data-platform/open-source-data-platforms/oci-data-flow/code-examples/run-oci-data-flow-from-functions.py:73]: No commented code in github please.

**Pull request merged**